com.ca.lsp.cobol/lsp-service-cobol/src/main/java/com/ca/lsp/cobol/model/ConfigurationSettingsStorable.java

```diff
@@ -25,10 +25,13 @@
  */
 @Value
 public class ConfigurationSettingsStorable {
-  private Object profiles;
   private List<String> paths;
 
+  public ConfigurationSettingsStorable(List<String> paths) {
+    this.paths = paths;
```

Comment:
I would prefer to have it wrapped with `Collections#unmodifiableList()` to ensure immutability.